script/runc.spec

```diff
@@ -0,0 +1,64 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _sbindir /usr/local/sbin
+%global _mandir  /usr/local/share/man
+%global _docdir  /usr/local/share/doc
+
+Name: runc
+Version: %{_version}
+Release: %{_release}
+Summary: A tool for managing OCI containers
+
+License: ASL 2.0
+Source: %{name}-%{version}.tar.gz
+
+URL: https://www.opencontainers.org/
+Vendor: OCI
+Packager: OCI 
+
+BuildRequires: gcc-go
+BuildRequires: glibc
+BuildRequires: make
+BuildRequires: libseccomp-devel 
+BuildRequires: libselinux-devel
+BuildRequires: libapparmor-devel
+BuildRequires: go-md2man
```

Comment:
@moypray sorry, please help me understand "release it anywhere by rpms". Do you mean runc upstream will host its own rpms like what docker upstream does?

Also, do you want to perhaps grab the fedora rpm spec itself and then add conditionals for CentOS, SUSE etc if you want a one spec fits all approach? See: http://pkgs.fedoraproject.org/cgit/rpms/runc.git/tree/runc.spec

As far as CentOS Virt SIG goes (for example: https://wiki.centos.org/Cloud/Docker), things will be changing quite a bit, in that we might be moving soon toward using unbundled golang deps for Fedora rebuilds, but we'll continue to use bundled deps for RHEL rebuilds. So, we might wanna add golang-deps in distro-based conditionals, see the Fedora spec file for example.

Also, I don't find libapparmor-devel anywhere on RHEL or Fedora. Does SUSE have it @cyphar ?